webtau-graphql/src/test/groovy/org/testingisdocumenting/webtau/graphql/GraphQLCoverageTest.groovy

```diff
@@ -20,15 +20,17 @@ import org.junit.Test
 import org.testingisdocumenting.webtau.utils.ResourceUtils
 import org.junit.Before
 
+import java.nio.file.Paths
+
 import static org.testingisdocumenting.webtau.graphql.TestUtils.validationResult
 
 class GraphQLCoverageTest {
     private GraphQLCoverage coverage
 
     @Before
     void setUp() {
-        def schemaUrl = ResourceUtils.resourceUrl('test-schema.graphql')
-        coverage = new GraphQLCoverage(new GraphQLSchema(schemaUrl.file))
+        def schemaUrl = Paths.get(ResourceUtils.resourceUrl('test-schema.graphql').toURI()).toString()
```

Comment:
trick to avoid `/C:\path`

Reply:
Which also reminds me, should we allow `http://` based location for schema?

Reply:
I'm actually about to rework how we gather the relevant information from the schema so this will change.